float32 length
float32 width
float32 height
uint32 id
geometry_msgs/PoseStamped pose_stamped


